Gopkg.toml

```diff
@@ -71,7 +75,7 @@
 
 [[constraint]]
   name = "github.com/mholt/archiver"
-  version = "2.1.0"
+  version = "3.1.0"
```

Comment:
This change will make us drop a local patch to the vendor tree: https://github.com/cloudfoundry-incubator/fissile/pull/414/files

We have an open issue to update this dependency once the upstream PR has been merged: https://github.com/cloudfoundry-incubator/fissile/issues/416

I will need to rebase my PR on top of a restructuring of the code from v2 to v3, and then still need to add additional code for Windows support before we can expect this to be merged.

So for now we should not updated our vendored version of mholt/archiver.

Reply:
ah.. I have to revert a bunch of code
the openshift package I used for streaming has symlink support, maybe we should consider switching if your PR is stuck